Add initial version of FAQ document

isabelcosta opened the pull request:
This PR is meant to add an initial version of our FAQ document, so we can start having this information accessible to members of the community.

Often some of these questions have to be answered over and over again, this will help mitigate some confusion. I don't plan this to be a full complete version, but the beginning in which we can add later.

Fixes #4 

These are some of the questions that were gathered in this [google document](https://docs.google.com/document/d/1w7xsO0wkiLNtqaiVIs4X7q54LeZ3vp3I6SWs5JZ7wy8/edit#). Check issue #4 to see all the members who contributed to it

vj-codes commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,94 @@
+A: No. AnitaB.org Open Source is a separate open source community which is not qualified for Google Developers Group or Women Tech Maker. Our community mainly focuses on providing opportunities to anyone to get involved in open source projects aligns with AnitaB.org missions and values.
+
+**Q: Where should I post about general questions?**
+A: If you have any questions on where we need help, reach out in #questions or #newcomers or attend one of the open sessions. There are updated in #announcements > Calendar Events.
```
There's #general stream too and adding links to them will aslo be better .
Last sentence will be "  **They** are updated in.... "

vj-codes commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,94 @@
+A: We rebranded from Systers Open Source to AnitaB.org Open Source in 2020. GitHub projects that were active or still needed development have been moved to [github.com/anitab-org](http://github.com/anitab-org). 
+
+**Q: Are we still using Systers-open source on Slack?**
+A: Nope, we moved to Zulip - [AnitaB-org.zulipchat.com](https://anitab-org.zulipchat.com/) and anyone who was in our Slack was invited to the new workspace. We are keeping track of Android Tasks here and Backend Tasks here that were created during Google Code-in 2019.
```
I think we have gone past the phase of keeping track of GCI tasks , all PRs and issues created during GCI are closed now on Github. The decision was taken during a PM session and May had aaproved closing all.
So the last sentence can be removed.

isabelcosta replied:
great suggestion!

vj-codes reviewed (changes requested):
@isabelcosta just suggested some small changes :)

isabelcosta commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,94 @@
+
+**Q: What is the minimal technical knowledge needed to contribute to AnitaB.org projects?**
+A: **Zero technical knowledge needed!** Because you can always help with documentation, testing the app, and creating wireframes to update it to the new branding guidelines.
+i.e. Reaching out to a therapist for PowerUp scenario content, reviewed UI designs for Wheelchairs from the Cerebral Palsy communities on Facebook.
```
Nice question, I think this came from the time we were maintaining the [PowerUp project](https://github.com/anitab-org/powerup-android), in which outreach examples of contributions were "obvious". Because as PowerUp was a game scenario type of app, to build the scenarios for pre adolescents, we could reach out to people with experience to ask for help building appropriate content. Now that we don't maintain that project, we could have other examples 🤔 I will take that example out, and then we can add as we see fit. @vj-codes

isabelcosta commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,94 @@
+**Q: Which Issues/PRs do I review as a Newcomer/Mentor/Maintainer?**
+A: All of them (regardless of the project).
+
+**Q: I am a past Google Code-in (GCI), participant. How can I contribute to the community?**
```
Will take that out! Thank you so much for mentioning this.
And I am sooooo glad too, that you came to our community through this program 🤗 🎉 You have been an amazing contributor! 👏 @vj-codes

isabelcosta commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,94 @@
+A: Check our calendar in different timezones: [PST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles) || [EST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/New_York) || [GMT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40gr ...
+
+**Q: I see a lot of projects have been archived. Which projects are active right now?**
+A: We have several active projects going on and need work: Mentorship System (Android, Backend, Flutter, iOS),  anitab-org.github.io, VMS, Portal. If you want to work on an archived project, you can fork it and work on it independently. Your contributions will still be recognised. 
```
They are active yes. I think this was supposed to be just examples, but for clarification, I will add more as you suggested :)

isabelcosta commented:
@vj-codes your suggestions were so spot on! 🎉 👏 💪 Thank you so much for doing this review! I addressed your comments, let me know if you see something else that can be improved ;)

aditmehta9 commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,90 @@
+A: Here are some reasons:
+- Private Messages were disabled.
+- Age minimum on Slack is 16 years old, which excludes our Google Code-in Participants.
+
```
Nothing big. 'Age minimum on Slack is 16 years old' can be written as 'Minimum age required to be on Slack is 16 years'.

isabelcosta replied:
done! thank you for your review @aditmehta9 🙌

isabelcosta commented:
@aditmehta9 @vj-codes could you please review this PR again?

aditmehta9 commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,90 @@
+
+**Q: I have found a bug. What should I do?**
+A: Raise an issue and wait for the approval. Once it’s approved please make it clear for others you are working on the issue.
+
```
Nothing big but on Grammarly, this sentence was shown more accurate - 'Once it's approved, please make it apparent for others you are working on that issue.'

vj-codes replied:
This makes it sound like the contributor has to make it clear that he/she working on the issue but generally following the community guidelines one has to be assigned to start working on a issue approval. 
So we can probably frame something which denotes that you can comment if you are interested to work on the issue and will be assigned soon in a bit proper English 
What say ? @isabelcosta @aditmehta

aditmehta9 replied:
Yes you are right. I agree.

isabelcosta replied:
nice point @vj-codes , I just changed it, let me know what you think :)

aditmehta9 commented on frequently-asked-questions.md:
```diff
@@ -0,0 +1,90 @@
+
+**Q: Can I contribute to GSoC projects even if I’m not a GSoC student?**
+A: Yes, GSoC projects are discussed in public channels because we want community involvement. You can even attend weekly meetings of each project if you want to! These projects are just as open source as they were before the GSoC selections. You can contribute with ideas, and even issues if they're not labeled as GSoC issues.
+
```
Here 'labeled' is used whereas on line number 37 'labelled' is used. Both are right 'labeled' is used in American English, and 'labelled' is used in British English.

isabelcosta replied:
have chosen labeled :) thank you so much for pointing that out

isabelcosta commented:
done with the changes @vj-codes @aditmehta9 💪

aditmehta9 reviewed (approved):
LGTM

vj-codes commented:
@isabelcosta VMS, portal and mentorship ios, flutter should be removed from active projects , only this change otherwise the doc looks great 🚀

isabelcosta commented:
@vj-codes can you take a look again please?

vj-codes reviewed (approved):
@isabelcosta it's perfect 🚀 
This doc will help newcomers a lot :)

isabelcosta commented:
Thank you so much for your thorough reviews @vj-codes @aditmehta9 🎉
